web-common/src/features/authentication/LocalAvatarButton.svelte

```diff
@@ -10,10 +10,15 @@
   } from "@rilldata/web-common/runtime-client/local-service";
   import Spinner from "@rilldata/web-common/features/entity-management/Spinner.svelte";
 
-  $: user = createLocalServiceGetCurrentUser();
+  $: user = createLocalServiceGetCurrentUser({
+    query: {
+      // refetch in case user does a login/logout from outside of rill developer UI
+      refetchOnWindowFocus: true,
+    },
+  });
   $: metadata = createLocalServiceGetMetadata();
 
-  $: loginUrl = `${$metadata.data?.loginUrl}/?redirect=${window.location.origin}${window.location.pathname}`;
+  $: loginUrl = `${$metadata.data?.loginUrl}?redirect=${window.location.origin}${window.location.pathname}`;
   $: logoutUrl = `${$metadata.data?.loginUrl}/logout?redirect=${$page.url.href}`;
```

Comment:
Perhaps using the `URL` constructor would be safer to avoid these sorts of issues, e.g.:
```
  $: loginUrl = new URL($metadata.data?.loginUrl);
  $: loginUrl.searchParams.set('redirect', `${window.location.origin}${window.location.pathname}`);
  
  $: logoutUrl = new URL($metadata.data?.loginUrl + '/logout');
  $: logoutUrl.searchParams.set('redirect', $page.url.href);
```